app/src/main/java/eu/kanade/tachiyomi/ui/browse/source/browse/BrowseSourceScreenModel.kt

```diff
@@ -238,7 +238,7 @@ class BrowseSourceScreenModel(
                 addTracks.bindEnhancedTrackers(manga, source)
             }
 
-            updateManga.await(new.toMangaUpdate())
+            updateManga.await(new.toMangaUpdate().copy(chapterFlags = null))
```

Comment:
Can't we just use `updateManga.awaitUpdateFavorite` here? It'll require a small amount a tweaking above.

Reply:
Actually looking at the implementation of `MangaScreenModel` the full implementation is different. Maybe try to sync them? Could make it a usecase and reuse in `HistoryScreenModel` to...